Lib/turtle.py

```diff
@@ -1289,7 +1282,6 @@ def _incrementudc(self):
         """Increment update counter."""
         if not TurtleScreen._RUNNING:
             TurtleScreen._RUNNING = True
-            raise Terminator
```

Comment:
Setting something just before terminating makes no sense.  But I am not sure what fix is best.

Reply:
Yes, it is not a very good idea, but it solves the problem. I am open to different solutions :)

Reply:
What problem?

Reply:
``
Python 3.8.2 (default, Apr  8 2020, 14:31:25) 
[GCC 9.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from turtle import Turtle
>>> t = Turtle()
>>> t.getscreen().bye()
>>>  t2 = Turtle()
  File "<stdin>", line 1
    t2 = Turtle()
    ^
IndentationError: unexpected indent
>>> t2 = Turtle()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.8/turtle.py", line 3813, in __init__
    RawTurtle.__init__(self, Turtle._screen,
  File "/usr/lib/python3.8/turtle.py", line 2557, in __init__
    self._update()
  File "/usr/lib/python3.8/turtle.py", line 2660, in _update
    self._update_data()
  File "/usr/lib/python3.8/turtle.py", line 2646, in _update_data
    self.screen._incrementudc()
  File "/usr/lib/python3.8/turtle.py", line 1292, in _incrementudc
    raise Terminator
turtle.Terminator
>>>